lib/instrument/transformer/nskeyed.js

```diff
@@ -272,10 +362,7 @@ class Unarchive {
   classForUid (index) {
     const meta = this.objects[index.UID];
     const name = meta.$classname;
-    const klass = UNARCHIVE_CLASS_MAP[name];
-    if (!klass) {
-      throw new Error(`no mapping for ${name} in UNARCHIVE_CLASS_MAP`);
-    }
+    const klass = UNARCHIVE_CLASS_MAP[name] || NSIgnore;
```

Comment:
perhaps replacing `||` with `??` would better show your intension

Reply:
yeah, that make sense